stylesheets/style.css

```diff
@@ -0,0 +1,44 @@
+
+html {
+    background: url(images/bg.jpg) no-repeat center center fixed;
+    background-size: cover;
+    height: 100%;
+    overflow: hidden;
+}
+body {
+    font-family:'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+    color: darkblue;
+    background-image: url("../images/blueridgebackground.jpg");
+    background-size: cover;
+    background-position: center;
+    background-attachment: fixed;
+    background-repeat: no-repeat;
+}
+
+h1 {
+    text-align: center;
+}
+
+h2 {
+    background: lightblue;
+}
+
+section.main {
+    background-color: rgba(156, 199, 213,.6);
+}
+
+nav ul {
+    margin: 0;
+    padding: 20px;
+    text-align: center;
+    list-style-type: none;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+    gap: 16px;
+    }
+nav li {
```

Comment:
I suggest keeping a single blank line between each CSS rule to make it easy to see at a glance where one rule set and an another begins.